modules/lang-painless/src/main/java/org/elasticsearch/painless/antlr/PainlessLexer.java

```diff
@@ -1,22 +1,17 @@
 // ANTLR GENERATED CODE: DO NOT EDIT
 package org.elasticsearch.painless.antlr;
 
+import org.antlr.v4.runtime.*;
```

Comment:
I *thought* we ran something that formatted these. I think we excuse these from the tests, but I thought we still had something to blow these open.

Reply:
The `regen` command in the painless should do it.

We used to run these through checkstyle and I turned that off years ago because we had to manually mess with the generated files just to add `{}` and stuff.

Reply:
But there's still some stuff in there. I think.

Reply:
As the imports are not related to this issue, please raise a separate PR and let's get this in.